.github/workflows/javascript.yml

```diff
@@ -12,9 +12,11 @@ jobs:
         with:
           node-version: 18
 
-      - name: Install standard
+      - name: Install standard and React plugin for ESlint
         run: |
-          npm install standard --location=global
+          npm install standard@17.1.2 --location=global
+          npm install eslint@9.10.0 --location=global
```

Comment:
Btw, the latest eslint version is 9.11.1 which just got released yesterday (https://www.npmjs.com/package/eslint). but i dont think this minor update matters :))